src/js_native_api_v8.cc

```diff
@@ -0,0 +1,2895 @@
+#include <limits.h>  // INT_MAX
+#include <cmath>
```

Comment:
I had to include `<algorithm>` for things to compile because of the usage of `std::min`. I think we should add this.

Reply:
Not sure what happened with this, but this isn't resolved for me. I still get compiler errors because of `std::min` usage.